tldr/cli.py

```
@@ -40,37 +41,9 @@ def _get_subprocess_detach_kwargs():
-    '.cs': 'csharp',
-    '.kt': 'kotlin',
-    '.kts': 'kotlin',
-    '.scala': 'scala',
-    '.sc': 'scala',
-    '.lua': 'lua',
-    '.luau': 'luau',
-    '.ex': 'elixir',
-    '.exs': 'elixir',
-}
+# Canonical list of supported languages for --lang choices (derived from semantic.py)
+LANG_CHOICES = ["auto"] + ALL_LANGUAGES
```

Ruff (0.15.6)

[warning] Consider `["auto", *ALL_LANGUAGES]` instead of concatenation

Replace with `["auto", *ALL_LANGUAGES]`

(RUF005)